build/src/transformTs.js

```diff
@@ -0,0 +1,186 @@
+const wr = require('../protos/rw/remote_pb')
+const https = require('https')
+const url = require('url')
+
+
+function attachSample(s, samples, metric_type = 'reg'){
```

Comment:
is it possible that the metric_type wont be one of (reg, sum, count, avg)?

Reply:
No, it's something I defined in the code in order to aggregate histogram metrics.